src/sentry/api/bases/project.py

```diff
@@ -123,42 +123,44 @@ def convert_args(
                 project = (
                     Project.objects.filter(
                         organization__slug__id_or_slug=organization_slug,
-                        slug__id_or_slug=project_slug,
+                        slug__id_or_slug=project_id_or_slug,
                     )
                     .select_related("organization")
                     .prefetch_related("teams")
                     .get()
                 )
             else:
                 project = (
-                    Project.objects.filter(organization__slug=organization_slug, slug=project_slug)
+                    Project.objects.filter(
+                        organization__slug=organization_slug, slug=project_id_or_slug
+                    )
                     .select_related("organization")
                     .prefetch_related("teams")
                     .get()
                 )
         except Project.DoesNotExist:
             try:
                 # Project may have been renamed
-                # This will only happen if the passed in project_slug is a slug and not an id
+                # This will only happen if the passed in project_id_or_slug is a slug and not an id
```

Comment:
:squint:

wait

Reply:
🥔

Reply:
Let's face it. This comment will rename confusing till removal of feature flag 😅

Reply:
yeah, once i remove the function, i think i can clean this up